### ISSUE TITLE
`JS` and `JSS` in uppercase and lowercase works in different ways

### ISSUE THREAD

**Issue by likemusic** [orientechnologies/orientdb#7782]
## Expected behavior  
Commands should be case-insensitive.
Resulf of ` JS print('Hello World!')` should be the same as ` js print('Hello World!')`.

## Actual behavior  
```
orientdb {db=StarterHabr}> JS print('Hello World!')
Hello World!

Client side script executed in 0,002000 sec(s). Value returned is: null
```

```
orientdb {db=StarterHabr}> js print('Hello World!')
[Started multi-line command. Type just 'end' to finish and execute]
end
Hello World!

Client side script executed in 0,002000 sec(s). Value returned is: null
```
Uppercase command `JS` and  `JSS` run the command and not goes to interactive javascript mode.
Lowercase version `js` and `jss` goes to interactive javascript mode.

## Steps to reproduce  
Type and run in console:
` JS print('Hello World!')`

` js print('Hello World!')`

**Comment by luigidellaquila** [comment]
Hi @likemusic 

Thank you very much for reporting, I just checked the docs and you are right, the console is not working as expected. I'm fixing it now

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @likemusic 

I just pushed a fix on 2.2.x, it will be available in v 2.2.29

Thanks

Luigi
